src/metabase/api/preview_embed.clj

```diff
@@ -37,6 +37,7 @@
       :card-id          card-id
       :token-params     (eu/get-in-unsigned-token-or-throw unsigned-token [:params])
       :embedding-params (eu/get-in-unsigned-token-or-throw unsigned-token [:_embedding_params])
+      :constraints      {:max-results 2000}
```

Comment:
One more thing: this is definitely a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) and it should go in a `def` somewhere rather than just living there right in the function call. As a bonus that would also let you use `with-redefs` to change it to make things easier to test, for example you can redefine it to 5 and make sure a table only returns 5 rows. That's easier (and a lot faster) than making sure a table (a) has more than 2000 rows and (b) returns the first 2000 rows. 

Even if you don't change the test to use `with-redefs`, the same magic number rule applies here. You should refer to the variable instead of hard-coding 2000. Why? Think about what happens if we change this number to something like 500 in the future. The test would start failing, and you'd have to go update the number there as well. Not a big deal here since it's only two places to update, but if we had say 30 places using the number we'd have 30 places to update and so forth. Better to avoid magic numbers for that reason (and also because it makes code more readable).